core/desugarDatatypes.ml

```diff
@@ -545,7 +545,7 @@ let sentence typing_env = function
   | Directive   d  -> typing_env, Directive d
 
 let read ~aliases s =
-  let dt, _ = Parse.parse_string ~in_context:(Parse.fresh_context ()) Parse.datatype s in
+  let dt, _ = Parse.parse_string ~in_context:(Parse.LinksLexer.fresh_context ()) Parse.datatype s in
```

Comment:
I'd probably define `let fresh_context () = LinksLexer.fresh_context ()` in `Parse`.

Reply:
`open Parse` seems to do a better job here.